src/paul_mclendahand/cmd_pmac.py

```diff
@@ -69,18 +51,20 @@ def get_config(args):
                 if key == "github_api_token":
                     # We don't let people set the api token in the setup.cfg
                     # file which gets checked in
-                    continue
+                    raise click.ClickException(
+                        "You shouldn't set github_api_token in setup.cfg."
+                    )
                 my_config[key] = config.get(key, "")
 
     # Override with environment variables
     for key in my_config.keys():
         if "PMAC_%s" % key.upper() in os.environ:
             my_config[key] = os.environ["PMAC_%s" % key.upper()]
 
-    # Override with command line arguments
-    for key in my_config.keys():
-        if getattr(args, key, ""):
-            my_config[key] = getattr(args, key)
```

Comment:
I couldn't figure out what this was about. It seems like it should allow for configuration to be set on the command line, but it doesn't seem to work. So I nixed it.